src/components/SingleRecipePage/SingleRecipePage.tsx

```diff
@@ -39,10 +39,11 @@ const SingleRecipePage = () => {
   const [recipe, setRecipe] = useState<SavedRecipe | null>(null);
   const [showConfirm, setShowConfirm] = useState(false);
   const [servingSize, setServingSize] = useState(0);
-  const [sendingIngredients, setSendingIngredients] = useState({});
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_sendingIngredients, setSendingIngredients] = useState({});
```

Comment:
Does somebody already plan on adding a loading indicator for this? If not, we should just remove it. I can do that if you want but wasn't sure what the plan was.

Reply:
Yes, this is for one feature that is waiting backend. I'll remove it we will not use it